### DISCUSSION TITLE
rendering of disabled outside days in range picker

### DISCUSSION THREAD

**Discussion by tameoo** [gpbl/react-day-picker#2575] [Support]
## Code 

```tsx
import { DayPicker } from "react-day-picker";

export function Start() {
  return (
    <DayPicker
        mode='range'
        numberOfMonths={3}
        selected={range}
        startMonth={fromMonth}
        endMonth={toMonth}
        month={month}
        fixedWeeks
        showOutsideDays
        weekStartsOn={1}
        onMonthChange={changeMonth}
        onSelect={changeRange}
        disabled={[
          {
            before: fromMonth,
            after: toMonth,
          },
        ]}
      />
  );
}
```

## Expected Behavior

it should add ouside days till 10th of november

## Actual Behavior

just stops on 7th november if i specified endMonth as 31of october

## Screenshots

this is my actual custom layout with responsive cells
![image](https://github.com/user-attachments/assets/dc929191-4168-4bf5-9aae-aa4e96aa204f)
![image](https://github.com/user-attachments/assets/8537d2e0-97f7-4d5e-ac0a-327b14f5eb9c)

it should be like this 
![image](https://github.com/user-attachments/assets/08379d05-d7f4-4feb-bef8-5ed5545659ba)

**Comment by gpbl** [comment]
Hey @tameoo, it seems like your code isn’t working. it is missing the declaration of the variables that you’re passing to the props. If you can share your code, I’ll try to reproduce the issue and help you out!
